monai/bundle/scripts.py

```diff
@@ -1019,6 +1053,148 @@ def ckpt_export(
     logger.info(f"exported to TorchScript file: {filepath_}.")
 
 
+def trt_export(
```

Comment:
I see most of the code in this `trt_export` is same as `trt_export`, so I think you can define a `_export(converter: callable)` function, then define `ckpt_export ` and `trt_export` to call `_export` with different `converter` callback. The converter can take `parser` and `model` as parameters.
What do you think?

Thanks.